### PR TITLE
[SQL Migration][New Migration Experience] Instance Summary created.

### PR DESCRIPTION
**Pull request by stuti149**
This PR:

- Creates Body Section for New Assessment Results Page.

- Body Section includes Tree Component (for list of instance and databases), middle section (for list of Warnings and Issues) and Instance Summary Section.

Mocks link:
https://www.figma.com/file/UaD2zc58Y2iGKxnqDFQY3Q/End-to-End-Design-GA---SQL-Migration-ADS?type=design&node-id=1510-76694&mode=design&t=GZRjG4olcxyBOacB-0

Screenshot:

image

### CODE REVIEW

File: extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+		const graph = this._view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'column',
+			}
+		}).component();
+
+		const labels = [
+			{
+				label: "Not ready",
```
Comment (brian-harris) [inline extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts:127]: can this be replaced with a resource string

Comment (stuti149) [reply extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts]: done.

File: extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts [hunk 2]
```diff
@@ -124,17 +124,17 @@ export class InstanceSummary {
 
 		const labels = [
 			{
-				label: "Not ready",
+				label: constants.NOT_READY,
 				count: 1,
 				color: "#E00B1C"
```
Comment (brian-harris) [inline extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts:129]: can these colors be replaced with named constants that reflect the color or purpose

Comment (stuti149) [reply extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts]: done

File: extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetialsBody.ts [hunk 3]
```diff
@@ -4,19 +4,86 @@
...
+				style: { "border": "solid 1px" }
+			},
+			options: [{ label: constants.SUMMARY_TITLE, id: "summary" }]
+		}).component();
+
+		assessmentFindingsComponent.addItem(findingsSection);
+
+		const warningsOrIssuesListSection = this._view.modelBuilder.listView().withProps({
+			title: { text: constants.WARNINGS },
+			options: []  // TODO: fill the list of options.
```
Comment (cpichuka) [inline extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetialsBody.ts:83]: Is it planned as part of this feature?

**Review by cpichuka — APPROVED** [review]

**Review by krritik — APPROVED** [review]

File: extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetialsBody.ts (hunk 3 above)
Comment (stuti149) [reply extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetialsBody.ts:83]: yes coming in next PR.

**Pull request merged**